XRTK-Core/Packages/com.xrtk.core/Providers/Controllers/BaseController.cs

```diff
@@ -81,6 +81,12 @@ protected BaseController(TrackingState trackingState, Handedness controllerHande
         /// <inheritdoc />
         public bool IsRotationAvailable { get; protected set; }
 
+        /// <inheritdoc />
+        public Vector3 AngularVelocity { get; protected set; }
+
+        /// <inheritdoc />
+        public Vector3 Velocity { get; protected set; }
```

Comment:
This was added to `BaseController` since `IMixedRealityController` requires it to be defined. See comment there for resolution.